L/libCEED/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libCEED"
+version = v"0.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/CEED/libCEED.git", "6134893e0067dc2a87bbbe22a551ae134ad5b885")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd libCEED
+sed -i "s/MEMCHK :=.*/MEMCHK := 0/g" Makefile 
+sed -i "s/CC_VENDOR :=.*/CC_VENDOR := gcc/g" Makefile 
+make -j$ncore
```

Comment:
This is simpler and doesn't require editing any file:
```suggestion
cd $WORKSPACE/srcdir/libCEED
make -j${nproc} MEMCHK=0 CC_VENDOR=cc
```
Also, the number of process is `${nproc}` :wink:

Reply:
Good simplifications, thanks.